<ui version="4.0" >
 <class>ConnectionWebServicesDialog</class>
 <widget class="QDialog" name="ConnectionWebServicesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WebServices Connection</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QLabel" name="m_labelConnection" >
     <property name="text" >
      <string>Send Request :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QProgressBar" name="m_sendProgressBar" >
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="m_abortPushButton" >
     <property name="text" >
      <string>Abort</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Receive Request :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QProgressBar" name="m_recieveProgressBar" >
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="m_labelState" >
     <property name="text" >
      <string>Connection ....</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
